src/components/Range/index.tsx

```diff
@@ -96,6 +97,13 @@ class Range extends Component<PropsType, StateType> {
         if (this.props.onChange !== undefined) this.props.onChange(inputValues());
     };
 
+    // tslint:disable-next-line
+    private handleChange = debounce(16, true, (values: RangeType): void => {
```

Comment:
16ms makes the dragging of the sliders feel janky. A value of 6 would be a better fit. 
```suggestion
    private handleChange = debounce(6, true, (values: RangeType): void => {
```